app/models/user.rb

```diff
@@ -84,6 +86,12 @@ def self.count_by_location(location, radius=20)
     near(location, radius.to_i)&.size
   end
 
+  def self.community_leaders_nearby(latitude, longitude, radius)
+    self
+      .near([latitude, longitude], radius)
+      .tagged_with('community-leader')
```

Comment:
You can pass in the the constant here for `LEADER`:

https://github.com/OperationCode/operationcode_backend/blob/master/app/models/user.rb#L2